doofinder/includes/class-multilanguage.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace Doofinder\WP;
+
+use Doofinder\WP\Multilanguage\I18n_Handler;
+use Doofinder\WP\Multilanguage\WPML;
+
+class Multilanguage {
+
+	/**
+	 * Singleton of this class.
+	 *
+	 * @var Multilanguage
+	 */
+	private static $_instance;
```

Comment:
Same as above

Reply:
Again, singleton pattern should declare the instance as static to be accessed from static methods like getInstance()